epsagon/triggers/aws_lambda.py

```diff
@@ -453,6 +454,60 @@ def __init__(self, start_time, event, context):
         }
 
 
+class CognitoLambdaTrigger(BaseLambdaTrigger):
+    """
+    Represents a Cognito Lambda Trigger
+    """
+    RESOURCE_TYPE = 'cognito'
+
+    def __init__(self, start_time, event, _context):
+        """
+        Initialize Trigger
+        :param start_time: event's start time (epoch unix)
+        :param event: event dict from entry point
+        :param context: context dict from entry point
+        """
+
+        super(CognitoLambdaTrigger, self).__init__(start_time)
+
+        self.event_id = 'cognito-{uid}'.format(uid=str(uuid4))
```

Comment:
Shouldn't you invoke the uuid4 function?

Reply:
wow good catch. indeed